api/node/Cargo.toml

```diff
@@ -27,6 +27,7 @@ slint-interpreter = { workspace = true, features = ["default", "display-diagnost
 spin_on = "0.1"
 css-color-parser2 = { workspace = true }
 itertools = { workspace = true }
+rand = "0.8.5"
```

Comment:
I'm curious why this adds rand

Reply:
Thanks for the hint. That is not needed.

Reply:
removed